src/token/Token.sol

```diff
@@ -242,12 +261,7 @@ contract Token is IToken, UUPS, Ownable, ReentrancyGuard, ERC721Votes, TokenStor
 
     /// @notice Burns a token that did not see any bids
     /// @param _tokenId The ERC-721 token id
-    function burn(uint256 _tokenId) external {
-        // Ensure the caller is the auction house
-        if (msg.sender != settings.auction) {
-            revert ONLY_AUCTION();
-        }
-
+    function burn(uint256 _tokenId) external onlyBurner {
```

Comment:
Do we have a design doc for this?

Not sure why we allow the auction to burn any token. It would make sense for me to allow only burns that are owned by the caller. 

Opening this burn type functionality owned by anyone to any address doesn't feel safe to me.

Reply:
Another way to solve this is `AllowedMinter`s can burn only tokens held by the minter address.

With this code if I make an EOA the burner, I can eject anyone from the DAO with that EOA.

Reply:
@iainnash any thoughts on what we should call this role? I think it's a good idea, but now "minter" seems insufficient if it can burn (in certain cases) as well. Maybe it's fine.

Reply:
Shouldn’t burning just allow only the current holder to burn?
Do we enable standard burning across the board for existing tokens @tbtstl ?

Reply:
this is outdated, see newest changes. its only minter that holds current token